yt_dlp/extractor/bilibili.py

```diff
@@ -502,39 +503,135 @@ def _real_extract(self, url):
             season_info.get('bangumi_title'), season_info.get('evaluate'))
 
 
-class BilibiliChannelIE(InfoExtractor):
-    _VALID_URL = r'https?://space.bilibili\.com/(?P<id>\d+)'
-    _API_URL = "https://api.bilibili.com/x/space/arc/search?mid=%s&pn=%d&jsonp=jsonp"
+def bilibili_space_extract_playlist(fetch_page, get_metadata, get_entries):
+    first_page = fetch_page(1)
+    metadata = get_metadata(first_page)
+
+    paged_list = InAdvancePagedList(
+        lambda idx: get_entries(fetch_page(idx) if idx else first_page),
+        metadata['page_count'], metadata['page_size'])
+
+    return metadata, paged_list
+
+
+class BilibiliSpaceVideoIE(InfoExtractor):
+    _VALID_URL = r'https?://space\.bilibili\.com/(?P<id>\d+)(?P<video>/video)?$'
```

Comment:
Because old version treat https://space.bilibili.com/3985676 as video only, I keep the old behavior, 
then BilibiliSpaceVideoIE and BilibiliSpaceIE are almost same (I add a info log for BilibiliSpaceIE),
so I merge them.

Should I split them?

Reply:
Mmm true, what do you think @pukkandan ?

Reply:
I should have translated the page 😶   TA的视频 means Ta's Videos, so it's okay to keep it like this

Reply:
most uploader only have videos by now, some also have audios or others, so I think old behavior should be kept.